### PR TITLE
Implemented observer pattern for driver change

### PR DESCRIPTION
**Pull request by milczarekmat**

### CODE REVIEW

File: src/main/java/edu/kis/powp/jobs2d/drivers/gui/DriverChangeObserver.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package edu.kis.powp.jobs2d.drivers.gui;
+
+import edu.kis.powp.jobs2d.features.DriverFeature;
+import edu.kis.powp.observer.Subscriber;
+
+public class DriverChangeObserver implements Subscriber {
```
Comment (kowallus) [inline src/main/java/edu/kis/powp/jobs2d/drivers/gui/DriverChangeObserver.java:6]: All DriverManager observers will be driver change observers. The name of observer should also describe its activity. Please rename.

**Review by kowallus — CHANGES_REQUESTED** [review]
Great! Please fix the name of the observer class to reflect its purpose.

File: src/main/java/edu/kis/powp/jobs2d/drivers/gui/updateDriverInfoObserver.java [hunk 2]
```diff
@@ -3,7 +3,7 @@
 import edu.kis.powp.jobs2d.features.DriverFeature;
 import edu.kis.powp.observer.Subscriber;
 
-public class DriverChangeObserver implements Subscriber {
+public class updateDriverInfoObserver implements Subscriber {
```
Comment (kowallus) [inline src/main/java/edu/kis/powp/jobs2d/drivers/gui/updateDriverInfoObserver.java:6]: Class name should start from UpperCase!

**Review by kowallus — CHANGES_REQUESTED** [review]
Fix name (java convention) and resolve conflicts (merge changes from other PR-s).

File: src/main/java/edu/kis/powp/jobs2d/drivers/gui/updateDriverInfoObserver.java (hunk 2 above)
Comment (Yibux) [reply src/main/java/edu/kis/powp/jobs2d/drivers/gui/updateDriverInfoObserver.java]: Class name renamed

**Review by Yibux — APPROVED** [review]

**Review by Yibux — APPROVED** [review]

**Review by kowallus — APPROVED** [review]

**Pull request merged**